maestro-test/src/test/resources/038_partial_id.yaml

```diff
@@ -4,5 +4,5 @@ appId: com.example.app
     id: "keyboard_area"
     retryTapIfNoChange: false
 - tapOn:
-    id: ".*keyboard_area"
+    id: ".*another_keyboard_area"
```

Comment:
Why was this changed?

Reply:
This was changed because we now match from depth first due to this change: https://github.com/mobile-dev-inc/maestro/pull/477/commits/bdad3e52090e95820adcfb4737331eb8b9552e87

Reply:
This doesn't seem right in this test case:

```
val driver = driver {
            element {
                id = "com.google.android.inputmethod.latin:id/another_keyboard_area"
                bounds = Bounds(0, 0, 100, 100)
            }

            element {
                id = "com.google.android.inputmethod.latin:id/keyboard_area"
                bounds = Bounds(0, 100, 100, 200)
            }
}
```

Those views are siblings, so I would've expected `id: .*keyboard_area` to match the first view

Reply:
FIxed ✅